utilities/operations/sites/restrictions.js

```diff
@@ -0,0 +1,65 @@
+const _ = require('lodash');
+const { mutedUserModel, blacklistModel, User } = require('models');
+
+exports.get = async ({ app }) => {
+  if (!app) return { result: {} };
+  const { result } = await mutedUserModel.find({ condition: { mutedForApps: _.get(app, 'host') }, sort: { _id: -1 } });
+  const { blackLists } = await blacklistModel.find({ user: { $in: [_.get(app, 'owner'), ..._.get(app, 'admins')] } });
+  const muted = _.uniqBy(result, 'userName');
+
+  const arrMutedNames = _.map(muted, 'userName');
```

Comment:
you can do _.uniq(_.map(muted, 'userName')) - it is easier to iterate over an array of strings for uniqueness than an array of objects, in uniqBy it will be harder to operate under the hood